praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
+## Context
+* You are the trusted committer or maintainer on an inner source project.
+* You value the community of contributors and want to maintain and grow it.
+
+## Forces
+You are busy, which makes it easy to forget some of the soft touches like praise and thanks.
+You may not be someone that is comfortable in social situations or good with words.
+
+## Solutions
+It feels good to anyone to be recognized by others.
+In a professional setting, increased recognition is also an avenue to increased influence and growth.
+Any time someone gives to your inner source project, recognize them with a sincere "thank you".
+
+For non-trivial contributions (all code contributions and also significant time contributions), say thank you via the following mechanisms:
+
+1. Call out the person by name in any chat location (e.g. _Slack_) where you organize your project activity.  Let everyone know what they did and thank them publicly.  Example:
+
+> Everyone @here give a high-five to @andrew.clegg for updating the _rcs-viewer_ to the latest version of the _hebo-client_ (https://github.com/rcs/rcs-viewer/pull/81).
+Thanks for helping keep this library up-to-date, Andy!
+
+2. Send an email to them and their manager (cc'd) privately thanking them for the contribution.
+For code contributions often-times you can just forward the merge notification mail.  Example:
+
+> Hi, Andy, I want to thank you again for making this update.
+It may have been a small amount of time, but it's attention like this from each person that make the RCS project work for all of us.
+Thanks for solving your own problem in a way that also makes the _rcs-viewer_ better for everyone.
+
+## Resulting Context
+Feedback like this leaves the contributor with a fantastic feeling and ready to come back for more.
+Combining **both** forms of thanks gives them recognition in front of their peers (breadth) and in front of their direct manager (depth).
+Additionally, there's also a subtle encouragement for those peers in chat to consider contributing themselves and for that manager to look for appropriate circumstances to encourage their other direct reports to do the same.
+
```

Comment:
It can also help with increasing awareness among managers about the value of your specific InnerSource project and of InnerSource as a method.

Reply:
From the text above I'm not sure, I can follow that argument chain "contributor spends time on inner source project, contributor gets thank you for that time [0] manager of contributor understands value of inner source project". I think there's one step missing where I've typed [0].

In open source the step in position [0] is the publicity generated from internal employees contributing to open source projects that implies benefits in company image resulting in a) benefits in recruiting new people and b) benefits in gaining more customers due to publicly shown open source engineering leadership. Is there an equivalent inside of companies?

Reply:
I understand your point, Isabel.  I think that the manager gets increased awareness about the project, but not necessarily its value.  It's very possible that the manager was completely unaware of the team's dependency on the project.  Now that awareness is there, but is it valuable?

People have brought up to me that looping in the manager could even be dangerous in that the manager may wonder why this developer is spending time contributing to another project?  In my experience that result hasn't really happened and most managers are excited to be effecting results outside their own team.  I think it gives credence to your point that this process isn't an automatic understanding of value.

I do think that project awareness does carry some value on its own, especially within a company, so I think it's worth mentioning.

Reply:
`Additionally, awareness of the inner source project is spread to the manager, who may have previously not known of the team's use and involvement with it`

Reply:
Wrt. to "looping in managers considered dangerous" ... I may be wrong here, but if that is a fear to me the situation the employee is in feels poisonous to me: It seems more of an environment where trust is a serious issue.